public/app/features/dashboard/services/TimeSrv.ts

```diff
@@ -77,13 +78,14 @@ export class TimeSrv {
     );
 
     if (range.to.isBefore(range.from)) {
-      this.setTime(
-        {
-          from: range.raw.to,
-          to: range.raw.from,
-        },
-        false
-      );
+      const appEvents = getAppEvents();
```

Comment:
Worth checking as well whether this works with Dashboards-as-scenes?

Enable the `scenes=true` feature flag and click the "View as Scene" button.

Reply:
As we changed this to not use a `Toast`, this compatibility check is no longer needed.